tembo-pgmq-python/README.md

```diff
@@ -121,8 +159,39 @@ print(popped_message)
 purged_count: int = queue.purge("my_queue")
 print(f"Purged {purged_count} messages from the queue.")
 ```
+
+### Detach an archive from a queue
+
+```python
+queue.detach_archive("my_queue")
+```
+
+### Drop a queue
+
+```python
+dropped: bool = queue.drop_queue("my_queue")
+print(f"Queue dropped: {dropped}")
+```
+
+### Validate the length of a queue name
+
+```python
+queue.validate_queue_name("my_queue")
+```
```

Comment:
This one doesn't need to be exposed by the python library, I think? WDYT, @ChuckHend? I can see it being useful for "dynamic names" but idk

Reply:
It is not an essential in the clients but I could see some people finding it useful.